src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -253,6 +259,13 @@ class BoundedExecutor {
   boost::asio::thread_pool pool_;
 };
 
+struct SchedulingQueueTag {
+  /// Worker ID for the caller.
+  WorkerID caller_worker_id;
+  /// Version for the caller.
+  int64_t caller_version = 0;
```

Comment:
It's probably clearer to just call this `caller_creation_timestamp_ms` and then specify in the comment that it's used as a version

Reply:
makes sense.

Reply:
makes sense. thanks